reference-code/src/main/kotlin/createcommands/Permissions.kt

```diff
@@ -74,6 +75,7 @@ fun permissions() {
     // /economy give <target> <amount> - requires the permission "economy.admin.give" to execute
     CommandAPICommand("economy")
         .withPermission("economy.admin.give") // The important part of this example
+        .withArguments(new LiteralArgument("give"))
```

Comment:
Kotlin doesn't have the `new` keyword https://kotlinlang.org/docs/classes.html#creating-instances-of-classes. They just use the class's name. Same comment applies to line 93.

```suggestion
        .withArguments(LiteralArgument("give"))
```

Reply:
oups, you're right